scripts/build-python-3.5.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+# Get the source
+mkdir -p /opt/sources
+cd /opt/sources
+wget -nv https://www.python.org/ftp/python/3.5.2/Python-3.5.2.tgz
+tar xzf Python-3.5.2.tgz
+
+# Build
+cd Python-3.5.2
+# Use /opt/python{X}.{Y} for the prefix.
+./configure --prefix=/opt/python3.5
+# Explicitly build the profile-guided-optimized interpreter
+make profile-opt
+make install
+
+# Clean-up sources
+rm /opt/sources/Python-3.5.2.tgz
+rm -r /opt/sources/Python-3.5.2
```

Comment:
1. If we do, not yet because this is still experimental.
2. `/usr/local` is intended for locally-built non-package-manager-managed software, so this would kind of fit the bill...
3. However, `/opt` can be used for the same purpose, but also for the purpose of pre-built software that is not package-manager-managed. So if we ever wanted to build the interpreter outside of this runtime we'd want it to be in `/opt`.

Overall, it doesn't matter to the user because `virtualenv` will handle manipulating the `PATH`. Where we put the interpreter is whatever.

(I vote for `/opt` until I have a good reason to integrate it into the rest of the FS)